src/test/java/com/rocana/lucene/codec/v1/TestRocanaSearchCodec.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2017 Rocana
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.rocana.lucene.codec.v1;
```

Comment:
the original version of this file, as it is in the main rocana repo, [is here](https://github.com/scalingdata/rocana/blob/a1dee63129aec1064516232eb9ccf88b15d0f250/rocana-search/src/test/java/com/rocana/search/server/index/codec/TestRocanaSearchCodecV1.java).